src/main/ank/1.5-products-and-coproducts.md

```diff
@@ -0,0 +1,275 @@
+```Haskell
+absurd :: Void -> a
+```
+```kotlin
+fun <A> absurd(): (Nothing) -> A
+```
+................
+```Haskell
+unit :: a -> ()
+unit _ = ()
+```
+```kotlin:ank:silent
+val unit: Unit = Unit
```

Comment:
This doesn't have the same signature in Kotlin as in Haskell, does it?

Reply:
for Types and Functions I used `val unit: () -> Unit = { }` - not sure if it's correct

Reply:
@LordRaydenMK `val` cannot be used with generics unless it specifies a generic receiver.